docs/monitors/collectd-processes.md

```diff
@@ -49,6 +49,78 @@ Monitor Type: `collectd/processes`
 
 
 
+## Metrics
+
+The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
+
+| Name | Type | Included | Description |
```

Comment:
This monitor needs to have `sendAll: true` to match existing behavior (it isn't in the whitelist).